### PR TITLE
docs: misc improved docs, added fetching raw events section

### PR DESCRIPTION
**Pull request by brayo-pip**
More updates.

----

| :rocket: | This description was created by [Ellipsis](https://www.ellipsis.dev) for commit 50e3a3a95fbce6d2735a570ee96b7f1c2087cfcb  | 
|--------|--------|

### Summary:
Updated documentation and examples for querying and fetching raw events in ActivityWatch, including version updates and a new example script.

**Key points**:
- Updated `src/api/rest.rst` to include a link to the Bucket REST API documentation.
- Updated version and release information in `src/conf.py` to `0.13` and `v0.13.1` respectively.
- Modified `src/examples/query_client.py` to use `client.client_hostname` instead of `client.hostname`.
- Added a new section on fetching raw events in `src/examples/querying-data.rst`.
- Added a new example script `src/examples/raw_events.py` to demonstrate fetching and modifying raw events.

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: src/importers.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Importers
 ActivityWatch can't track everything, so sometimes you might want to import data from another source.
 
 - :gh-aw:`aw-import-ical`, supports importing from ``.ical`` files or syncing with Google Calendar.
-- :gh-aw:`aw-importer-smartertime`, imports from `smartertime`_ (Android time tracker).
+- :gh-aw:`aw-importer-smartertime`, imports from `smartertime`_ (Deprecated in favour of - :gh-aw:`aw-android`).
```
Comment (ErikBjare) [inline src/importers.rst:7]: I don't know if "deprecated" is the right term. I still use it while we're working on making aw-android more accurate :)

File: src/examples/raw_events.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+for event in events:
+    app = event.data["app"]
+    if app in time_spent:
+        time_spent[app] += event.duration
+    else:
+        time_spent[app] = event.duration
+
+print(time_spent)
+
+# Rename app "Code" to "VS Code" in the events
```
Comment (ErikBjare) [inline src/examples/raw_events.py:26]: I don't think this section belongs in this example. 

Replacing events is something you'd rarely want to do, and we could just refer to the relevant aw-client example for redacting events.

File: src/importers.rst (hunk 1 above)
Comment (brayo-pip) [reply src/importers.rst:7]: What would be a better way to phrase it?

File: src/examples/raw_events.py (hunk 2 above)
Comment (brayo-pip) [reply src/examples/raw_events.py:26]: Cool we redo this bit.

**Comment by brayo-pip** [comment]
@ErikBjare Anything else you think that I should add?

**Review by ErikBjare — APPROVED** [review]
It's great, sorry for the delay!

**Pull request merged**